src/python/pants/core/deprecated_v1_target_types.py

```diff
@@ -104,6 +108,14 @@ class BundlesField(SequenceField):
     expected_element_type = Bundle
     expected_type_description = "an iterable of `bundle` objects"
 
+    @classmethod
+    def compute_value(cls, raw_value: Optional[Iterable[Bundle]], *, address: Address) -> None:
+        try:
```

Comment:
you can use [contextlib.suppress](https://docs.python.org/3/library/contextlib.html#contextlib.suppress)

Reply:
Oh, good point. This code gets is only being added to 2.0.0, though, in a hackily named `deprecated_v1_target_types.py` file, so I'm not gonna worry about it.

Good to know for the future, though.